libraries/helpers.rb

```diff
@@ -44,13 +44,19 @@ def pdns_package_module_requirements
       required_packages << 'pdns-backend-pgsql'
     when 'gmysql'
       required_packages << 'pdns-backend-mysql'
+    when 'random'
+      # Random isn't available on Ubuntu
```

Comment:
You can put a `next` in there under the comment to jump to the next loop iteration.